node/src/accountant/mod.rs

```diff
@@ -1252,7 +1241,7 @@ mod tests {
             PayableDaoMock::new().non_pending_payables_result(vec![payable_account.clone()]);
         let subject = AccountantBuilder::default()
             .bootstrapper_config(config)
-            .payable_daos(vec![PayableScannerDest(payable_dao)])
+            .payable_daos(vec![ToPayableScanner(payable_dao)])
```

Comment:
I think the ToXxxScanner (or whatever) name is fine; if you're still not satisfied with it, you might try ForXxxScanner; it kind of fits with the rest of the code when it's called.